associacao-brasileira-de-normas-tecnicas-ufrgs-initials.csl

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="pt-BR">
   <info>
-    <title>Universidade Federal do Rio Grande do Sul - ABNT (autoria abreviada) (Portuguese - Brazil)</title>
-    <title-short>UFRGS-ABNT</title-short>
+    <title>Universidade Federal do Rio Grande do Sul - SBUFRGS - ABNT (autoria abreviada) (Portuguese - Brazil)</title>
```

Comment:
What's the SB for? that's not in your other styles, and it looks like UFRGS is the standard acronym for the university. Rest looks good

Reply:
It's for "Sistema de Bibliotecas", but UFRGS is better. I atualized in both styles (#4521 and here).
Thanks!